src/service/extensions-impl.js

```diff
@@ -271,27 +265,21 @@ export class Extensions {
    * @param {string} extensionId
    * @param {boolean=} includeInserted If true, includes script elements that
    *   are inserted by the runtime dynamically. Default is true.
-   * @return {?{el: !Element, attr: (string|undefined)}}
+   * @return {?Element}
    * @private
    */
   getExtensionScript_(extensionId, includeInserted = true) {
     const attr = this.attributeForExtension_(extensionId);
-    const modifier = includeInserted ? '' : ':not([i-amphtml-inserted])';
+    // Always ignore <script> elements that have a mismatched RTV.
+    const modifier =
+      ':not([i-amphtml-loaded-new-version])' +
+      (includeInserted ? '' : ':not([i-amphtml-inserted])');
     const {head} = this.win.document;
-    let el = head./*OK*/ querySelector(
-      `script[${attr}="${extensionId}"]` + modifier
-    );
-    if (el) {
-      return {el, attr};
-    }
-    // Some extensions don't have an attribute e.g. amp-viewer-integration.
-    el = head./*OK*/ querySelector(
-      `script[src*="/${extensionId}-"]` + modifier
-    );
-    if (el) {
-      return {el};
-    }
-    return null;
+    const selectors = [
+      `script[${attr}="${extensionId}"]` + modifier,
+      `script[src*="/${extensionId}-"]` + modifier,
```

Comment:
If we want to, we can use this as the only query.

Reply:
Yea that's probably fine.